test/spec/modules/conversantAnalyticsAdapter_spec.js

```diff
@@ -0,0 +1,733 @@
+import sinon from 'sinon';
+import {expect} from 'chai';
+import {getGlobal} from 'src/prebidGlobal.js';
+import conversantAnalytics from 'modules/conversantAnalyticsAdapter.js';
+import * as caObj from 'modules/conversantAnalyticsAdapter.js';
+import * as utils from 'src/utils.js';
+import * as prebidGlobal from 'src/prebidGlobal';
+import events from 'src/events';
+import constants from 'src/constants.json'
+
+describe('Conversant analytics adapter tests', function() {
+  let sandbox; // sinon sandbox to make restoring all stubbed objects eaiser
+  let xhr; // xhr stub from sinon for capturing data sent via ajax
+  let clock; // clock stub from sinon to mock our cache cleanup interval
+
+  const PREBID_VERSION = '1.2';
+  const SITE_ID = 108060;
+
+  let requests = [];
```

Comment:
This can be const.  JS const just means the reference can't change, but the object can.  In this case, requests must point to the same array, but the content of the array can be changed.